packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -306,6 +306,20 @@ export abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 	constructor( config: EditorConfig = {} ) {
 		super();
 
+		if ( !isPlainObject( config ) ) {
+			/**
+			 * Editor configuration must be a plain JavaScript object.
```

Comment:
Not true, at least not in the way how I understand this. POJO for me isn't just about top-level but also other levels. And we have e.g. also callbacks or elements in our config if we look deeper.

So, this error message may be confusing, IMO.

Reply:
@niegowski nope https://github.com/toss/es-toolkit/blob/main/src/predicate/isPlainObject.ts